CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ project(httpd-datadog)
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+set(CMAKE_CXX_OUTPUT_EXTENSION_REPLACE ON)
```

Comment:
[question] Mind explaining this addition?

Reply:
I had a problem with generating the coverage and it could fix the issue. In fact, it was not the right solution. It is an oversight, i removed it with my new commit